script.js

```diff
@@ -128,39 +130,34 @@ class MixOrMatch {
         });
     }
     getCardType(card) {
-        let cardFront = card.children[1];
-        return cardFront.children[cardFront.children.length-1].src;
+        return card.getElementsByClassName('card-value')[0].src;

```

Comment:
I changed this to use the class name applied to all the card values since it will not break that way if the HTML changes, and it is easier to read and understand.